index.js

```diff
@@ -35,7 +35,9 @@ async function fastifyHelmet (fastify, options) {
 
   fastify.addHook('onRequest', async (request, reply) => {
     /* istanbul ignore next */
-    const { helmet: routeOptions } = request.routeOptions.config || request.context.config
+    const { helmet: routeOptions } = request.routeOptions.config ||
+    request.routeConfig ||
```

Comment:
why keep `request.context.config`?

Reply:
I would follow this fix to avoid the same issue: https://github.com/fastify/fastify-static/pull/410

Reply:
But they already check for both of them, kibertoad is saying that the third check is unnecessary